scripts/exrrfs_lbc.sh

```diff
@@ -3,10 +3,10 @@ declare -rx PS4='+ $(basename ${BASH_SOURCE[0]:-${FUNCNAME[0]:-"Unknown"}})[${LI
 set -x
 cpreq=${cpreq:-cpreq}
 if [[ -z "${ENS_INDEX}" ]]; then
-  prefixin=${LBC_PREFIX:-LBC_PREFIX_not_defined}
+  prefixin=${EXTRN_MDL_SOURCE:-LBC_PREFIX_not_defined}
   ensindexstr=""
 else
-  prefixin=${ENS_LBC_PREFIX:-ENS_LBC_PREFIX_not_defined}
+  prefixin=${EXTRN_MDL_SOURCE:-ENS_LBC_PREFIX_not_defined}
```

Comment:
Line 6 and 9: 
The part after - needs to modified to EXTRN_MDL_SOURCE_not_defined to match the new situation.

BTW, I don't use "" because I found that when we check the log file, the "not_defined" text will be more verbose to indicate what's wrong.

Reply:
Changed